### PR TITLE
[BUGFIX] Use relative imports for capitalone_dataprofiler_expectations package

### PR DESCRIPTION
**Pull request by kenwade4**
Fixes all the capitalone_dataprofiler_expectations that fail on import with `ModuleNotFoundError: No module named 'contrib'`

Changes proposed in this pull request:
- Move get_table_columns_metric func to great_expectations.expectations.metrics.util
- Use relative imports in capitalone_dataprofiler_expectations

Tested with

```
pytest -vv contrib/capitalone_dataprofiler_expectations

build-gallery --backends "pandas" expect_column_values_confidence_for_data_label_to_be_greater_than_or_equal_to_threshold expect_column_values_confidence_for_data_label_to_be_less_than_or_equal_to_threshold expect_column_values_to_be_equal_to_or_greater_than_profile_min expect_column_values_to_be_equal_to_or_less_than_profile_max expect_column_values_to_be_probabilistically_greater_than_or_equal_to_threshold expect_profile_numeric_columns_diff_between_exclusive_threshold_range expect_profile_numeric_columns_diff_between_inclusive_threshold_range expect_profile_numeric_columns_diff_greater_than_or_equal_to_threshold expect_profile_numeric_columns_diff_greater_than_threshold expect_profile_numeric_columns_diff_less_than_or_equal_to_threshold expect_profile_numeric_columns_diff_less_than_threshold expect_profile_numeric_columns_percent_diff_between_exclusive_threshold_range expect_profile_numeric_columns_percent_diff_between_inclusive_threshold_range expect_profile_numeric_columns_percent_diff_greater_than_or_equal_to_threshold expect_profile_numeric_columns_percent_diff_greater_than_threshold expect_profile_numeric_columns_percent_diff_less_than_or_equal_to_threshold expect_profile_numeric_columns_percent_diff_less_than_threshold
```

### CODE REVIEW

File: great_expectations/expectations/metrics/util.py [hunk 1]
```diff
@@ -123,6 +125,37 @@
     import sqlalchemy  # noqa: TID251
 
 
+def get_table_columns_metric(engine: ExecutionEngine) -> [MetricConfiguration, dict]:  # type: ignore[valid-type]
```
Comment (Kilo59) [inline great_expectations/expectations/metrics/util.py]:
```suggestion
def get_table_columns_metric(engine: ExecutionEngine) -> tuple[MetricConfiguration, dict]:
```

**Review by anthonyburdi — CHANGES_REQUESTED** [review]
Let's try to find a way to not have to revert to using relative imports. It's difficult to read and also goes against our style guide https://google.github.io/styleguide/pyguide.html#22-imports.

**Comment by kenwade4** [comment]
Every other Expectation package is using relative imports

**Comment by anthonyburdi** [comment]
> Every other Expectation package is using relative imports

I'm not seeing that - I've looked through a couple dozen expectations in several packages. Could you point me to some so I can understand?

**Pull request closed without merging**